lib/node_modules/@stdlib/stats/base/dists/normal/README.md

```diff
@@ -115,6 +115,96 @@ var normal = require( '@stdlib/stats/base/dists/normal' );
 console.log( objectKeys( normal ) );
 ```
 
+```javascript
```

Comment:
Prefer instead to modify the existing example. Project conventions are not to separate out examples into several consecutive Markdown code blocks.

Reply:
Merged all blocks, and did the same for example js files

Reply:
@kgryte What else do I need for this PR?